src/mptcpize.c

```diff
@@ -136,6 +136,9 @@ static char *locate_unit(const char *name)
 		if (strncmp(line, SYSTEMD_UNIT_VAR, strlen(SYSTEMD_UNIT_VAR)) == 0) {
 			char *ret = strdup(&line[strlen(SYSTEMD_UNIT_VAR)]);
 
```

Comment:
(I see in the code around that usually no new line is added between the allocation and the check but that's a microscopic detail ;-) )

Reply:
Good coding style catch.  Commit 282cc6f takes remove the blank line between the allocation and subsequent `NULL` check.